src/views/Pools/components/PoolsTable/PoolsTable.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { useRef } from 'react'
+import styled from 'styled-components'
+import { Button, ChevronUpIcon } from '@pancakeswap/uikit'
+import { useTranslation } from 'contexts/Localization'
+import { Pool } from 'state/types'
+import PoolRow from './PoolRow'
+
+interface PoolsTableProps {
+  pools: Pool[]
+  userDataLoaded: boolean
+  cakeVault?: Pool
+  account: string
+  showFinishedPools: boolean
+}
+
+const StyledTable = styled.div`
+  box-shadow: ${({ theme }) =>
+    theme.isDark
+      ? `rgba(217, 217, 226, 0.05) 0px 1px 0px, rgba(217, 217, 226, 0.3) 0px 0px 8px;`
+      : `rgba(17, 17, 26, 0.05) 0px 1px 0px, rgba(17, 17, 26, 0.1) 0px 0px 8px;`}
```

Comment:
Do we need to add this to the UI Kit? If so please add a TODO

Reply:
yeah those are not in UIKit, will confirm with Bugs exact shadows and put it there 👍

Reply:
actually talked with Bugs, we're ditching shadows to outlines, will rework this